### DISCUSSION TITLE
My download is capped at 50 posts and i don't know if doing something wrong please help.

### DISCUSSION THREAD

**Discussion by mukeshmukesh00** [Serene-Arc/bulk-downloader-for-reddit#833] [Q&A]
this is the line i'm using:
python.exe -m bdfr clone C:\folder\data\ --user me --saved --authenticate  --file-scheme '{POSTID}

it also ends on this line:
DuplicateReplaceException: A duplicate comment has been detected. Are you attempting to call 'replace_more_comments' more than once?

**Comment by Serene-Arc** [comment]
You should open an issue and supply the logs.
